### ISSUE TITLE
Breaks post listing view on latest WordPress.

### ISSUE THREAD

**Issue by Lewiscowles1986** [dshanske/simple-location#177]
Hello David,

I received the following WordPress error message via email

```
Howdy!

Since WordPress 5.2 there is a built-in feature that detects when a plugin or theme causes a fatal error on your site, and notifies you with this automated email.

In this case, WordPress caught an error with one of your plugins, Simple Location.

... (omitted generic fluff)

When seeking help with this issue, you may be asked for some of the following information:
WordPress version 5.4.2
Current theme: Twenty Twenty (version 1.2)
Current plugin: Simple Location (version 4.1.5)
PHP version 7.2.30



Error Details
=============
An error of type E_ERROR was caused in line 149 of the file /home/*****/public/wp-content/plugins/simple-location/includes/class-loc-view.php. Error message: Uncaught Error: Call to a member function set() on null in /home/*****/public/wp-content/plugins/simple-location/includes/class-loc-view.php:149
Stack trace:
#0 /home/*****/public/wp-content/plugins/simple-location/includes/class-loc-view.php(336): Loc_View::get_location()
#1 /home/*****/public/wp-includes/class-wp-hook.php(287): Loc_View::location_content('blah blah, h...')
#2 /home/*****/public/wp-includes/plugin.php(206): WP_Hook->apply_filters('blah blah, h...', Array)
#3 /home/*****/public/wp-includes/formatting.php(3823): apply_filters('the_content', '\nblah blah, ...')
#4 /home/*****/public/wp-includes/class-wp-hook.php(287): wp_trim_excerpt('\nblah blah, ...', Object(WP_Post))
#5 /home/*****/public/wp-includes/plugin.php(206): WP_Hook->apply_filters('', Array)
#6 /home/***** (message is truncated)
```

I did disable the plugin after the admin post listing view refused to list posts with the plugin enabled. I only just noticed this error message.

It looks like a null guard is needed for some reason. Maybe I disabled an API key this needs and other errors are being swallowed? Anyway I thought you should know. It's not a live site, so this is unlikely to be a high-priority issue.

**Comment by dshanske** [comment]
Odd...I didn't touch that file. Will investigate

**Comment by Lewiscowles1986** [comment]
I've added a PR for the minimal case fix. It's sloppy.

Should use specific encapsulated exception which is caught at source, so that specific handlers can be added at the WordPress integration layer to raise notifications etc on platform. Instead I borrowed a behavior from above and switched out a variable. It's more opaque than I'd like, but does the job.

That said, it does stop configuration taking out sites.
